Source/Krypton Components/Krypton.Ribbon/General/DesignTimeDraw.cs

```diff
@@ -47,21 +47,21 @@ internal class DesignTimeDraw
         /// <param name="context">Rendering context.</param>
         /// <param name="clientRect">Client rectangle of the source view.</param>
         /// <param name="state">State of element.</param>
-        public static void DrawArea(KryptonRibbon ribbon,
+        public static void DrawArea(KryptonRibbon? ribbon,
                                     RenderContext context,
                                     Rectangle clientRect,
                                     PaletteState state)
         {
             Color c = state == PaletteState.Normal
-                ? ribbon.StateCommon.RibbonGeneral.GetRibbonGroupSeparatorDark(PaletteState.Normal)
-                : ribbon.StateCommon.RibbonGroupButton.Back.GetBackColor1(PaletteState.Tracking);
+                ? ribbon!.StateCommon.RibbonGeneral.GetRibbonGroupSeparatorDark(PaletteState.Normal)
```

Comment:
if `ribbon == null` what will the value of `c` be ?

Reply:
So, a check if `ribbon != null`?

Reply:
Resolved :)